gprofiler/profilers/java.py

```diff
@@ -746,8 +866,15 @@ def __init__(
         self._pid_to_java_version: Dict[int, Optional[str]] = {}
         self._kernel_messages_provider = get_kernel_messages_provider()
         self._enabled_proc_events_java = False
+        self._java_collect_jvm_flags = self._init_java_flag_collection(java_collect_jvm_flags)
```

Comment:
`_java` prefix is not needed (needed for the args because it's the same namespace with other args, but this object is `JavaProfiler`, note that no other attr has `_java_` prefix